srcpkgs/cudatext-qt5/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'cudatext-qt5'
+pkgname=cudatext-qt5
+version=1.168.6.1
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz python3"
+depends="qt5 python3"
+short_desc="Cross-platform text and code editor, written in Lazarus"
+maintainer="luan-adao <luanadao98@gmail.com>"
+license="MPL-2.0"
+homepage="https://cudatext.github.io"
+distfiles="https://sourceforge.net/projects/cudatext/files/release/${version}/cudatext_${version}-1_qt5_amd64.deb"
```

Comment:
as already mentioned by @hervyqa , please build from source, if source is available we don't accept repackaging like this